tests/std/tests/P1135R6_atomic_flag_test/test.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <atomic>
+#include <cassert>
+#include <execution>
+#include <numeric>
+#include <vector>
+
+constexpr auto mo = std::memory_order_seq_cst;
+
+
+template <typename FlagType, typename IsSet, typename TestAndSet, typename Clear>
+void test_flags(const IsSet is_set, const TestAndSet test_and_set, const Clear clear) {
+    constexpr std::size_t unique      = 800;
+    constexpr std::size_t repetitions = 800;
+    constexpr std::size_t total       = unique * repetitions;
+    constexpr std::size_t dups        = total - unique;
+
+    FlagType flags[unique];
+    std::vector<FlagType*> ptrs;
+    for (std::size_t i = 0; i != repetitions; i++) {
+        for (std::size_t j = 0; j != unique; j++) {
+            ptrs.push_back(&flags[j]);
+        }
+    }
+
+    const auto& par = std::execution::par;

```

Comment:
I think a using-declaration would be simpler (so we wouldn't have to think about a distinct reference):
```suggestion
    using std::execution::par;
```